src/interface/mesh_data.hpp

```diff
@@ -290,6 +290,14 @@ class MeshData {
 
   const auto &GetAllBlockData() const { return block_data_; }
 
+  bool ContainsGid(int gid) const {
```

Comment:
Any concern about stale `gid`s  here (c.f., https://github.com/parthenon-hpc-lab/parthenon/pull/1010)?

Reply:
I don't think so, since the `MeshData` should be cleared and recreated on remeshing. This doesn't rely on the gids of the variables themselves.